orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/MonitorWebhookTask.groovy

```diff
@@ -73,6 +78,10 @@ class MonitorWebhookTask implements Task {
         return new TaskResult(ExecutionStatus.TERMINAL,
           [error: [reason: e.message, response: response.body]])
       }
+
+      if (isSingleElementCollection(progress))
```

Comment:
Support for collection with single element, to handle JSONPath returning list as response